hydrophone.go

```diff
@@ -69,12 +69,21 @@ func main() {
 		config.Ses.Region = "us-west-2"
 	}
 
+	config.Mongo.FromEnv()
+
 	// server secret may be passed via a separate env variable to accomodate easy secrets injection via Kubernetes
 	serverSecret, found := os.LookupEnv("SERVER_SECRET")
 	if found {
 		config.ShorelineConfig.Secret = serverSecret
 		config.Api.ServerSecret = serverSecret
 	}
+
+	protocol, found := os.LookupEnv("PROTOCOL")
+	if found {
+		config.Api.Protocol = protocol
+	} else {
+		config.Api.Protocol = "https"
```

Comment:
Will we need some certificates for the rest of the stack ?

Reply:
This env is about the protocol used by the main API (i.e. yourloops). It is only used to generate href attributes in html email templates (see getWebURL function and WebURL key in templates).

The config of http/https for the service is stored in config.Service.Scheme (loaded from TIDEPOOL_HYDROPHONE_SERVICE env).